selfdrive/updated.py

```diff
@@ -96,10 +70,8 @@ def run(cmd, cwd=None):
 def remove_consistent_flag():
   os.system("sync")
   consistent_file = Path(os.path.join(FINALIZED, ".overlay_consistent"))
-  try:
+  if os.path.isfile(consistent_file):
```

Comment:
`if consistent_file.exists()`?

Once we switch to python 3.8 we can use `consistent_file.unlink(missing_ok=True)` :)